contrib/jenkins_tests/rpm.sh

```diff
@@ -88,5 +88,40 @@ if [ $opt_checkpkg -eq 1 ]; then
     test_id=$((test_id+1))
 fi
 
+
+# check if we have email of indevidual users in the packages rpm/deb metadata Maintainer field
+pacakges_location="$rpm_dir"/dist-pkg/packages
+email_log_file="$rpm_dir"/dist-pkg/email_scan.log
+
+if [ $opt_rpm -eq 1 ]; then
+    search_filter="*.rpm"
+    test_info_exec="rpm -qpi --changelog"
+else
+    search_filter="*.deb"
+    test_info_exec="apt info"
+fi
+
+# iterate on all packages and extarct the metadata to outout file
+find "$pacakges_location" -type f -name "$search_filter" -exec $test_info_exec {} \; |& tee -a "$email_log_file"
+
+do_archive "$email_log_file"
+
+set +e
+# grep email strings exclude allowed email networking-support@nvidia.com
+test_output=$(grep -E -o "\b[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Za-z]{2,6}\b" "$email_log_file" | grep -v "networking-support")
+test_rc=$?
+# check rc - grep will return 0 if it found such mail and 1 if not
+if [[ $test_rc -eq 0 ]]; then
```

Comment:
With this approach you'll find if there is  a bad email, but it'll be not clear which RPM/DEB has is
Same code can be executed per found package and if bad email found, clear error message can be printed

Reply:
Its fine it will be in the log available in artifacts and in any way the email is usually mentioned in 1 source file (for all packages), so its kind of obvious where the problem is